ironfish/src/wallet/walletdb/multisigIdentityValue.ts

```diff
@@ -19,6 +19,7 @@ export interface MultisigIdentityValue {
    * while allowing distinction between undefined and actual secrets during deserialization.
    */
   secret?: Buffer
+  ledger: boolean
```

Comment:
Since this field isn't optional we need a migration to update existing stored identities.

Reply:
I think we may actually want to avoid adding `ledger` to this datastore. I think we should avoid importing just the identity from a ledger device without also importing a full account, and the goal of https://github.com/iron-fish/ironfish/pull/5692 is to not have multisig identities stored for imported accounts